definitions/mock/ibc.mock.proto

```diff
@@ -1,19 +1,25 @@
 syntax = "proto3";
 package ibc.mock;
 
+import "google/protobuf/duration.proto";
+import "google/protobuf/timestamp.proto";
 import "ibc/core/client/v1/client.proto";
 
 message Header {
   ibc.core.client.v1.Height height = 1;
-  uint64 timestamp = 2; 
+  google.protobuf.Timestamp timestamp = 2;
 }
 
 message ClientState {
   Header header = 1;
+  google.protobuf.Duration trusing_period = 2;
+  google.protobuf.Duration max_clock_drift = 3;
```

Comment:
IMO, `trusting_period` and `max_clock_drift` is more of client specific fields, let’s keep the mock structure as general as possible and if some tests cannot be covered by mock, instead having them for tendermint `ClientState`.

Reply:
I wanted to model client expiration - that's why I wanted to add `trusting_period`.

I guess we can remove `max_clock_drift` but implying `max_clock_drift` is zero - as in client updates with future timestamp are invalid.